### DISCUSSION TITLE
Was expecting the custom_function.md to be with the .sql file...

### DISCUSSION THREAD

**Discussion by kirkw** [okbob/plpgsql_check#129] [General]
Pavel, I was expecting this .md file to be stored in the examples/ folder...
It's your stuff... But I figured it would be more likely to be read.  And if we have 10-12 examples, we might have 10-12 .md files.  Not to clutter up the main folder?

**Comment by okbob** [comment]
ne 25. 9. 2022 v 8:48 odesílatel Kirk Wolak ***@***.***>
napsal:

> Pavel, I was expecting this .md file to be stored in the examples/
> folder...
> It's your stuff... But I figured it would be more likely to be read. And
> if we have 10-12 examples, we might have 10-12 .md files. Not to clutter up
> the main folder?
>

I don't see this to be a problem.  Just there should be correct links from
README.md

**Comment by hadeersaeyd651-glitch** [comment]
> Pavel, I was expecting this .md file to be stored in the examples/ folder...
> It's your stuff... But I figured it would be more likely to be read.  And if we have 10-12 examples, we might have 10-12 .md files.  Not to clutter up the main folder?
